src/framework/components/element/image-element.js

```diff
@@ -994,7 +1007,8 @@ class ImageElement {
         if (this._materialAsset) {
             const asset = this._system.app.assets.get(this._materialAsset);
             if (!asset || asset.resource !== value) {
-                this.materialAsset = null;
```

Comment:
This created a circular call back to `set material` where it gets set to null

Reply:
It was to remove the events, but wasn't thinking of the circular callback, nice spot!